packages/script-editor/src/ScriptEditor.tsx

```diff
@@ -53,6 +56,7 @@ const OUTPUT_AREA_CHILD_CLASS = 'elyra-ScriptEditor-OutputArea-child';
 const OUTPUT_AREA_OUTPUT_CLASS = 'elyra-ScriptEditor-OutputArea-output';
 const OUTPUT_AREA_PROMPT_CLASS = 'elyra-ScriptEditor-OutputArea-prompt';
 const RUN_BUTTON_CLASS = 'elyra-ScriptEditor-Run';
+// const RUN_AND_DEBUG_BUTTON_CLASS = 'elyra-ScriptEditor-Run-Debug';
```

Comment:
why is this commented out? should it be removed?

Reply:
This should be removed, still from first implementation approach